app/assets/stylesheets/common.css.scss

```diff
@@ -35,8 +35,21 @@ html,body {
     background-color: $primary;
   }
 
-  section, aside {
-    margin: 0px 16px 32px 16px;
+  section {
+    margin: 16px;
+  }
+
+  aside {
+    margin: 0px 16px 16px 16px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0px 16px 16px` instead of `0px 16px 16px 16px`<br>`0px` should be written without units as `0`